action/images.py

```diff
@@ -20,6 +20,10 @@
 from tarfile import open as tarfile_open
 from shutil import copytree
 from pexpect import spawn as px_spawn, TIMEOUT as px_TIMEOUT
+from re import match as re_match
+from requests import HTTPError as requests_HTTPError
+from dockersave import Image as DockersaveImage
+from json import loads as json_loads
```

Comment:
Why do this? Why not just `import re`, etc? I know you did this earlier, but I missed it then.

Reply:
Ok, as we discussed, it's been this way before you started work on this. We'll have to switch to regular imports, I don't see a good reason to use these aliases.